docs/topics/http/views.txt

```diff
@@ -122,14 +122,14 @@ Example usage::
             raise Http404("Poll does not exist")
         return render_to_response('polls/detail.html', {'poll': p})
 
-In order to use the ``Http404`` exception to its fullest, you should create a
-template that is displayed when a 404 error is raised. This template should be
-called ``404.html`` and located in the top level of your template tree.
-
-If you provide a message when raising an ``Http404`` exception, it will appear
-in the standard 404 template displayed when :setting:`DEBUG` is ``True``. Use
-these messages for debugging purposes; they generally aren't suitable for use
-in a production 404 template.
+In order to show customized HTML when a 404 is encountered, you can create a
```

Comment:
Actually I had "when Django returns a 404" and @kezabelle suggested using "encountered".  I don't mind either way, thought I should just mention that first though :)